src/app/policies/views/PolicyListView.vue

```diff
@@ -150,7 +150,7 @@
                                 >
                                   <template #icon>
                                     <KIcon
-                                      color="var(--black-400)"
+                                      :color="KUI_COLOR_BACKGROUND_NEUTRAL_STRONGER"
```

Comment:
issue: you should use color-text tokens for svg colors 

Comment applies to whole PR